stablehlo/tests/interpret_outfeed.mlir

```diff
@@ -0,0 +1,19 @@
+// RUN: stablehlo-translate --interpret -split-input-file %s
+
+module @distribution_ops {
+  func.func public @outfeed(%inputs0 : tensor<2x2x2xi64>, %token : !stablehlo.token) -> !stablehlo.token {
+    %result = "stablehlo.outfeed"(%inputs0, %token) {
+      outfeed_config = ""
```

Comment:
If outfeed_config is empty, we probably shouldn't be specifying it in the first place?